### PR TITLE
Fix for generic eth price usage

### PR DESCRIPTION
**Pull request by jinchung**
When using generic ETH asset for pricing, use the other ethereumUtils function

### CODE REVIEW

**Review by brunobar79 — APPROVED** [review]
🙏

**Comment by brunobar79** [comment]
Let's hold on merging this

File: src/utils/ethereumUtils.js [hunk 1]
```diff
@@ -25,13 +25,18 @@ import {
...
 const { RNBip39 } = NativeModules;
 
-const getEthPriceUnit = genericAssets => {
-  return genericAssets?.eth?.price?.value || 0;
+const getEthPriceUnit = () => {
+  const { assets, genericAssets } = store.getState().data;
+  const genericEthPrice = genericAssets?.eth?.price?.value;
+  if (genericEthPrice) return genericEthPrice;
+  const ethInWallet = getAsset(assets);
+  return ethInWallet?.price?.value || 0;
```
Comment (osdnk) [inline src/utils/ethereumUtils.js:39]:
```suggestion
  return genericEthPrice || getAsset(assets)?.price?.value || 0;
```

**Review by osdnk — APPROVED** [review]

**Review by skylarbarrera — APPROVED** [review]
LGTM 🙏

**Pull request merged**